tests/test-data/keys/ecdsa_prime256v1.pem

```diff
@@ -0,0 +1,5 @@
+-----BEGIN EC PRIVATE KEY-----
+MHcCAQEEIKE0vihID+ffiAB2O8DW7d9kTRcGHA7a7bsr0lu2NPaKoAoGCCqGSM49
+AwEHoUQDQgAEc563pICgt7T27Hu1nnJ1Qis6jwIEx0ixBihKkeJRurqz/2EZtpmi
+o263hkYfxpgr2T0dvqu/qfSS5Jbc14Iw3A==
+-----END EC PRIVATE KEY-----
```

Comment:
There's no reason for the key to be in PEM format, is there? We tolerate the Chromium tests being in PEM format because we want to keep them in sync with Chromium's tree, but for our own test vectors I'd prefer to avoid unnecessary base64 encoding. OpenSSL tools always (AFAICT) provide a way to output binary files instead of PEM, usually with `-outform der` or similar.